packages/gatsby-source-drupal/src/utils.js

```diff
@@ -1,19 +1,108 @@
 const _ = require(`lodash`)
+const axios = require(`axios`)
+
 const {
   nodeFromData,
   downloadFile,
   isFileNode,
+  getHref,
   createNodeIdWithVersion,
 } = require(`./normalize`)
 
+const { getOptions } = require(`./plugin-options`)
+
 const backRefsNamesLookup = new WeakMap()
 const referencedNodesLookup = new WeakMap()
 
+const fetchLanguageConfig = async ({
+  translation,
+  baseUrl,
+  apiBase,
+  basicAuth,
+  headers,
+  params,
+  reporter,
+}) => {
+  const defaultLanguageConfig = {
```

Comment:
This could all be passed via options instead of being resolved during the build. It would require documentation but means less code and more flexibility. Requiring a Drupal user is especially impactful on larger sites which would benefit greatly from edge caching like Varnish but are forced to bypass it because they are using Basic Auth to access language config.

Reply:
Ooo yeah, let's just drop this. It's a small time saver (and nothing breaks without as the default language would at least still work) but slowing down all sourcing forever is much much worse.

Reply:
implemented in https://github.com/gatsbyjs/gatsby/pull/26720/commits/a518bcafbd6c61c90ac093bc6a31fe0333117efb